DSCResources/cChocoPackageInstallerSet/cChocoPackageInstallerSet.schema.psm1

```diff
@@ -33,13 +33,18 @@ Composite DSC Resource allowing you to specify multiple choco packages in a sing
         [ValidateNotNullOrEmpty()]
         [System.String]
         $Source
+        [parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]
+        [bool]

```

Comment:
To keep with the rest of the data types in the parameter declaration can you use [System.Boolean]?